packages/sdk/hardhat.config.cjs

```diff
@@ -1,5 +1,16 @@
+require('dotenv').config();
 require('@nomicfoundation/hardhat-toolbox-viem');
-
 module.exports = {
-  solidity: '0.8.24',
+  networks: {
```

Comment:
not sure if it matters, but if you only wanted to use this configuration if there's an alchemy key you could do
```ts
  ...(process.env.VITE_ALCHEMY_API_KEY ? {
    hardhat: {
      chainId: 8453, // Base mainnet chain ID
      //hardfork: 'cancun',
      forking: {
        url:
          'https://base-mainnet.g.alchemy.com/v2/' +
          process.env.VITE_ALCHEMY_API_KEY,
      },
    },
  } : {})
```